Expensify/App
# Quick cancel bugfix

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7089

### Tests



1. Edit a message.
2. Write something and hit ESC or click cancel button within 1 second.
3. See editing component doesn't re-opening.

### QA Steps


1. Edit a message.
2. Write something and hit ESC or click cancel button within 1 second.
3. See editing component doesn't re-opening.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/1279379/149817643-7a281297-735f-49fb-acd0-90b0c1216cc9.mp4


#### Mobile Web


https://user-images.githubusercontent.com/1279379/149817650-fb2fad5e-6e0c-42a3-ab10-4d5a5373c982.mp4


#### Desktop


#### iOS


#### Android


https://user-images.githubusercontent.com/1279379/149818434-88d699b0-402d-4e5e-b90c-0ac14ef58f23.mp4
